plugins/arElasticSearchPlugin/lib/model/arElasticSearchActorPdo.class.php

```diff
@@ -349,6 +355,31 @@ public function serializeAltNames()
     return $serialized;
   }
 
+  public function serializeObjectRelations()
+  {
+    $relations = array();
+
+    $sql = "SELECT r.type_id, r.object_id, r.subject_id, oo.class_name \r
+              AS object_class, so.class_name AS subject_class \r
+              FROM ".QubitRelation::TABLE_NAME." r \r
+              INNER JOIN ".QubitObject::TABLE_NAME." oo ON r.object_id=oo.id \r
+              INNER JOIN ".QubitObject::TABLE_NAME." so ON r.subject_id=so.id \r
+              WHERE r.object_id=? OR r.subject_id=?";
+
+    foreach (QubitPdo::fetchAll($sql, array($this->id, $this->id)) as $relation)
+    {
+      array_push($relations, array(
+        'typeId'       => $relation->type_id,
+        'objectId'     => $relation->object_id,
+        'objectClass'  => $relation->object_class,
+        'subjectId'    => $relation->subject_id,
+        'subjectClass' => $relation->subject_class
+      ));
+    }
+
+    return $relations;
```

Comment:
If you change the SELECT statement to `SELECT r.type_id as typeId, r.object_id as objectId, r.subject_id as subjectId FROM ...`, you could return the array directly with:

```php
return QubitPdo::fetchAll($sql, array($this->id, $this->id), array('fetchMode' => PDO::FETCH_ASSOC));
```

Not a big deal in here but `FETCH_ASSOC` reduces memory usage and it will return quicker without the loop.